### PR TITLE
Docs Sentry Dart Plugin

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/flutter/upload-debug.mdx [hunk 1]
```diff
@@ -20,6 +20,10 @@ For Flutter Desktop (Windows/macOS/Linux) `split-debug-info` and `obfuscate` fla
 
 Errors raised from the native layer in Flutter apps require certain debug information files to be uploaded. For example, an Android app can use `proguard` for minification and obfuscation. And when using NDK, dwarf debug files need to be uploaded. Flutter Web requires sourcemaps and iOS apps also require dwarf debug information files.
 
+## Sentry Dart Plugin
+
+[Try out the Alpha version of the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin) that does it automatically for you, or follow the manual process below.
```
Comment (marandaneto) [inline src/platforms/flutter/upload-debug.mdx:25]:
```suggestion
[Try out the Alpha version of the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin) that does it automatically for you, or follow the manual steps below.
```

or steps?

Comment (imatwawana) [reply src/platforms/flutter/upload-debug.mdx:25]:
```suggestion
[Try out the Alpha version of the Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin), which uploads debug symbols automatically for you, or follow the manual steps below.
```

**Review by imatwawana — APPROVED** [review]
Wording suggestion, otherwise LGTM!

File: src/platforms/flutter/upload-debug.mdx (hunk 1 above)
Comment (marandaneto) [reply src/platforms/flutter/upload-debug.mdx]: thanks :)

**Pull request merged**